### PR TITLE
a/b test of sort order

### PR DESCRIPTION
**Pull request by owocki**
##### Description

splits users into two groups, one that sees weighted shuffle grants + another that sees random grants (based upon even/odd-ness of their PK).

this will be evaluated by blockscience down the line, so they can see what effect gitcoin's default sort order has on outcomes

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/7792

##### Testing

tested locally

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -538,6 +538,9 @@ def get_grants(request):
     page = request.GET.get('page', 1)
     collections_page = request.GET.get('collections_page', 1)
     sort = request.GET.get('sort_option', 'weighted_shuffle')
+    if request.user.is_authenticated and request.user.profile.pk % 2 == 1:
+            if sort == 'weighted_shuffle':
+                sort = 'random_shuffle'
```
Comment (aamustapha) [inline app/grants/views.py:543]: The indentation here looks off.

**Review by aamustapha — CHANGES_REQUESTED** [review]
LGTM. Just one

**Review by chibie — APPROVED** [review]
only note @aamustapha 's comment

File: app/grants/views.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/views.py:543]: Yup it def is off ! @owocki you sure this worked locally ?  I'm fixing it up for now cause i need everything in round 8 branch 

nice catch @aamustapha

**Comment by thelostone-mc** [comment]
fixed up indent + redid migrations + refactord code

**Pull request merged**